Tools/PostProcessing/plot_nci_growth_rate.ipynb

```diff
@@ -0,0 +1,370 @@
42\ndkZtDChiabUm7hKVboVFLO2CyXy7ZM+w4vhqHL7ndDaddBRqYV++MELVDT0+YZe/hLoX3zep4x7m\n+XpjsWfi8nsNwsUyacIQpmfheDbeqSreZFVd4g1YvMjGhTuexnP38JzymFMei8PPhGQ41PoOc+ab\nONc41AAPP1e3Q8/Q3otLcivrPLSZiNycg8VkYkdEaWG0LG6gNJkOF/9eQ2eG472arBsSRXcwjwuH\ny8u9k8OqxfndJcomZWM3p64TkrymWE2wOqxDafVir2k7rNuoY1m08aLhKxo3HmmMc6XDAuYGJwn3\n7w5ucbpIEv7mPDXqbsru8YSk6uApjFZDfK36TrkUkuGkXdPEqkHWbugv73JXf8DxTii1dnG3x9n1\nZertMLrm3Yp2J9R1Ho5S2Bkv+g6LHdNRuCbUeUKVZ2yTU1Qpo07G8e6QY+0hpztbdJJqkqgebw84\nkQ/YKHIu7vTY2cmpt8IiyjACGqbd1V2n6tY0faOd1aSt0EPeSht6eYF1R2RpqGWdmLNdd1ive/SS\nglHTmtQDHtZt1ssu60WX84M+GztdRoMWXichGY+VI5oqxFXKMDI2nquexJFFqxOq0ijaLbw3opuV\nrLaHnGpvsZIOSa2hxig9YddTdpoORROm0pWDNowSysQZ9Dps7na4sNznrnxnsjPhetHlwrDH1jCn\nqhKKdspmJ+d8ucyF1hK5lTzYvsS7uwM6dp5z9Ys8t3oXXx3czwsbb+EbF06x9d0lsksZrU3DhxlV\nL4k3dTF+78braVy4f2CKy0fCUU6Sr7fDy5s8+NbTPP9Hv3HbGyVy1NkffPjgP6RhPbmxA8VkgLff\ne5rnf0dxWW6vHwYeu43Hvxv46fi4Vfak4vKd6CgnyTdkZo8Dj8dPR2b20izbM2UngfOzbsQU6Xxv\n3n0HefIWl/75Gf/bkzf59EV6DeQWKC4v1N+IzvfmKS7fgY5ykny9HV4AiHuTfxrAzJ6/3raM80bn\nO9+meb7u/t5p/B65490wJoPiss53fikuL55b3CtxKiY7vJhZm1BU+ukZt0lEZFEpJovIQjmyPck3\n2OFFRESmSDFZRBbNkU2S4do7vFzDp29nW44gne98W7TzlTvAAWMyLN77WOc73xbtfBeeubajFRER\nERG5wlGekywiIiIiMhNzkSSb2XvN7GUz+6aZfXTW7bndzOxVM/uamb1gZs/Puj2Hzcz+zMzO7S0d\nZWYnzOwLZvZK/Hd1lm08TNc4398zs9fja/yCmf3ULNsochCLFpNBcVlxWebRHZ8k79kq9SeBB4EP\nmNmDs23VVPyYu6/NafmdzwL7y998FHjW3d8GPBs/nxef5c3nC/CJ+BqvxbmgIkfeAsdkUFxWXJa5\ncscnyezZKtXdC2C8Varcodz934CL+778s8Dn4sefA9431UbdRtc4X5E7lWLyHFJclkU0D0ny1bZK\nvXdGbZkWB54xsy/H3a0WwWl3fyN+fAY4PcvGTMmHzOzFOOw3N8OYMvcWMSaD4rLissydeUiSF9GP\nuPsaYTjzg2b2o7Nu0DR5KMky72VZPgk8AKwBbwAfn21zROQGFJcVl2XOzEOSfFNbpc4Td389/nsO\neIowvDnvzprZPQDx33Mzbs9t5e5n3b129wb4DIvxGst8WLiYDIrLissyj+YhSV6orVLNrG9my+OP\ngfcAL13/p+bC08Cj8eNHgb+fYVtuu/GFJ/o5FuM1lvmwUDEZFJfjx4rLMneO9I57N2MBt0o9DTxl\nZhBev79y93+abZMOl5n9NfAIcNLMXgN+F3gS+Bsz+1Xg28D7Z9fCw3WN833EzNYIw5evAr82swaK\nHMACxmRQXFZclrmkHfdERERERPaZh+kWIiIiIiKHSkmyiIiIiMg+SpJFRERERPZRkiwiIiIiso+S\nZBERERGRfZQki4iIiIjsoyRZRERERGQfJclyXWb2pJl9YdbtEBGRQHFZZDqUJMuNrAEvzLoRIiIy\nobgsMgVKkuVG1oCvzroRIiIyobgsMgVKkuWazOxu4DSxx8LM+mb2eTP7ipndfwjH/4iZ+VUev///\nPbaIyDxSXBaZHiXJcj1rwBB42cy+H/gPoALe5e6vHsLxPwncs+fxceAM8OeHcGwRkXmkuCwyJUqS\n5XrWgK8B7wO+BHzG3X/J3YeHcXB333L3M+5+BngU+ADwiLt/8zCOLyIyhxSXRabE3H3WbZAjysw+\nD7wHSIGfcfd/PaTjpu5e7/n8t4EPAj/u7t84jN8hIjKPFJdFpkc9yXI9a8DfAS3gxP5vmtlTcS7c\nc2b2LTP7oWsdyMyeNrM/NrPngF/Z8/WPEQLxIwrEIiI3pLgsMiVKkuWqzKwHvA34E+Ax4C/M7B37\nnvYw8BV3fyfwBPCb1znkQ8DL7v5Od//T+DueAB4H3q2hPBGR61NcFpkuJclyLQ8DDrzk7n8JfAL4\nBzO7F8DMloCcsKgD4L+B1asdyMyWgdTd/3DP1z4G/DrwC8COmd0dH/ntOiERkTuc4rLIFClJlmtZ\nA17ZsxjkCeCLwNOxN+Mh4L/2zGF7B2ExydW8nbDABAAzM+C3gLviMd/Y83jXIZ+HiMi8UFwWmaJs\n1g2Qo8ndPwV8as/nDrx//LmZPQzcZ2Yt4BjwIeDn4/eeBX7Z3V+PT3+IPYE6Hmvldp+DiMg8UVwW\nmS4lyXKrHgb+EfgyYMBH3P1bsTfie4GLe577EPDM9JsoIrJQFJdFDpFKwMktMbN/B37R3b+z7+s/\nCDzm7h+eTctERBaT4rLI4VKSLLfEzP7H3R+YdTtERCRQXBY5XEqSRURERET2UXULEREREZF9lCSL\niIiIiOyjJFlEREREZB8lySIiIiIi+yhJFhERERHZR0myiIiIiMg+SpJFRERERPZRkiwiIiIiss//\nAQwt3pGm0mS6AAAAAElFTkSuQmCC\n",
```

Comment:
Can you please clear the cells before committing this file?
We just want to avoid that the source repo has to store too large files.